x-pack/plugins/beats/server/routes/api/register_verify_beats_route.js

```diff
@@ -49,7 +49,7 @@ async function verifyBeats(callWithRequest, beatIds) {
   return get(response, 'items', []);
 }
 
-function findNonExistentBeatIds(beatsFromEs, beatIdsFromRequest) {
+function determineNonExistentBeatIds(beatsFromEs, beatIdsFromRequest) {
```

Comment:
`determine*` turned out to be better after all

Reply:
Ugh, sorry, this wasn't intentional but a result of conflict fixing. Thanks for catching it, I'll fix it.